src/rdf4cpp/rdf/datatypes/registry/DatatypeConversion.hpp

```diff
@@ -91,6 +93,10 @@ consteval ConversionLayer auto make_conversion_layer_impl(LayerAcc const &table_
                 inline static typename target_type::cpp_type convert(typename source_type::cpp_type const &value) noexcept {
```

Comment:
Please resolve the TODO in the method documentation (LayerAcc template parameter)